### PR TITLE
CAN-5

### PR DESCRIPTION
**Pull request by dpegasusm**
Add CircleCI file for concatenation and deployment.

Also breaks the canvas file into two separate parts Google Analytics and Canvas Code

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+          command: |
+            echo 'export AWS_ACCESS_KEY_ID="$CANVAS_S3_AWS_KEY"' >> $BASH_ENV
+            echo 'export AWS_SECRET_ACCESS_KEY="$CANVAS_S3_AWS_SECRET"' >> $BASH_ENV
+            echo 'export AWS_REGION="us-east-1"' >> $BASH_ENV
+            source $BASH_ENV
+      - aws-s3/copy:
+          from: build/theme.min.css
+          to: 's3://hsph-canvas'
+      - aws-s3/copy:
+          from: build/theme.css
```
Comment (guillaumemolter) [inline .circleci/config.yml:52]: Any reason why are moving the unpacked version?

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+workflows:
+  version: 2
+  build-stage:
+    jobs:   
+      - canvasbuild:
+          context: org-global
+          filters:
+            branches:
+              only:
+              - CAN-5
```
Comment (guillaumemolter) [inline .circleci/config.yml]: We probably don't want to use this branch for all deployements

File: package.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
+{
+    "private": true,
+    "scripts": {
+      "dev": "NODE_ENV=development node_modules/webpack/bin/webpack.js --progress --hide-modules",
+      "watch": "NODE_ENV=development node_modules/webpack/bin/webpack.js --watch --progress --hide-modules",
+      "hot": "NODE_ENV=development webpack-dev-server --inline --hot",
```
Comment (guillaumemolter) [inline package.json]: What are these?

File: package.json [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+      "production": "NODE_ENV=production node_modules/webpack/bin/webpack.js --progress --hide-modules",
+      "eslint": "eslint resources/assets/js/** webpack.mix.js --fix --ext .js"
+    },
+    "devDependencies": {
+      "jquery": "^3.2"
```
Comment (guillaumemolter) [inline package.json]: jQuery should probably be an actual dependency

**Review by guillaumemolter — CHANGES_REQUESTED** [review]
I'm a little confused, as most of the scope of CAN-5 is not included in the PR. And CircleCI config was not in the scope of CAN-5...

**Comment by dpegasusm** [comment]
Im trying to break this into smaller parts because its taking a while rather than have one infinitely long PR and task. Theres 85 Commits in this one and the next few are probably going to be similar.

File: .circleci/config.yml (hunk 1 above)
Comment (dpegasusm) [reply .circleci/config.yml:52]: In case something goes wrong with the packing and we need them to see things. It cant hurt.

**Pull request closed without merging**